recipes/djinni-support-lib/all/conanfile.py

```diff
@@ -28,14 +28,14 @@ class DjinniSuppotLib(ConanFile):
     @property
     def objc_support(self):
         if self.options.target == "auto":
-            return self.settings.os in ["iOS", "Macos"]
+            return self.settings.os in ["iOS", "Macos", "watchOS", "tvOS"]
```

Comment:
```suggestion
            return tools.is_apple_os(self.settings.os): 
```

Reply:
The function "is_apple_os" check Macos too, we cannot use it.

Reply:
your code does exactly the same (macos was here before your PR actually), so I don't understand what you are referring to

Reply:
we can use `tools.is_apple_os` for the check if objective-C shall be enabled, not for the JNI part, because if you build of Macos, there is not reason why JNI should be disabled, if that should be the case, jni should explicit set.

@paulo-coutinho , can you please change the part for the `objc_support` to use `tools.is_apple_os` ?
the second suggesting, `not in ["iOS", "watchOS", "tvOS"]` for the jni check is also very correct, good catch Eric

Reply:
that's @prince-chrismc's eagle eye, not mine !

Reply:
fixed

Reply:
> that's @prince-chrismc's eagle eye, not mine !

You are both awesome reviewers Eric! 
Thanks a lot @prince-chrismc !